tests/end2end/features/utilcmds.feature

```diff
@@ -0,0 +1,70 @@
+Feature: Miscellaneous utility commands exposed to the user.
```

Comment:
There are various tests for things in `utilcmds.py` in `misc.feature` already (like `:message-*` or `:run-with-count`, and probably more) - could you move them here too?